Update getting started (and some other changes)

bramkragten opened the pull request:
## Proposed change

Updates the getting started page to explain the different installation methods better.

Some general style changes to align everything better (border radius, borders, shadows)
Fixes in energy landing page


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

bramkragten committed:
Update getting started (and some other chnages))

bramkragten committed:
fix + format

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,559 @@
+
+  <h2>Plug and play with Home Assistant Green</h2>
+  <p>
+    The affordable Home Assistant Green is the easiest way you can start using
```
```suggestion
    The affordable Home Assistant Green is the easiest way to start using
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,559 @@
+            <div>
+              <b>TOOLS REQUIRED</b>
+              <ul>
+                <li>Raspberry Pi 3 / 4 / or newer</li>
```
```suggestion
                <li>Raspberry Pi 3 or 4 with power supply</li>
```
removing newer, making a note to add it back in as soon as Pi 5 is supported

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,559 @@
+    Home Assistant can repurpose and be installed on all sorts of hardware, such
+    as ODroid and generic x86-64 machines. Home Assistant Operating System
+    allows you to install Home Assistant on these devices even if you have
+    little to no Linux experience.
```
```suggestion
    Home Assistant can be repurposed and installed on various hardware, such
    as an Odroid or a generic x86-64 machine. The Home Assistant Operating System
    allows you to install Home Assistant on these devices even if you have
    little to no Linux experience.
```

bramkragten committed:
Make images square

bramkragten committed:
Update images

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,559 @@
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Basic smart home setup skills</li>
```
```suggestion
                <li>Interest in setting up a smart home</li>
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,559 @@
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Very basic electronics skills akin to building a PC</li>
+                <li>Flashing a Raspberry Pi</li>
```
```suggestion
                <li>You're comfortable doing the following tasks based on instructions:</li>
                <li>Installing a compute module and a heat sink</li>
                <li>Flashing a Raspberry Pi</li>
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,559 @@
+          <img src="/images/installation/odroid.jpg" />
+        </div>
+        <div class="content">
+          <h3>Install Home Assistant on ODroid devices</h3>
```
```suggestion
          <h3>Install Home Assistant on Odroid devices</h3>
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,559 @@
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Understanding of hardware requirements</li>
+                <li>Writing boot images to your boot medium</li>
```
```suggestion
                <li>You're comfortable doing the following tasks based on instructions:</li>
                <li>Writing boot images to your boot medium</li>
                <li>Installing an SD card or eMMC</li>
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,559 @@
+              <b>TOOLS REQUIRED</b>
+              <ul>
+                <li>An Odroid device</li>
+                <li>MicroSD card</li>
```
```suggestion
                <li>MicroSD card or eMMC</li>
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,559 @@
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Building a PC or formatting a PC from scratch</li>
+                <li>Writing boot images to your boot medium</li>
```
```suggestion
                <li>You can use a command line and install a boot medium on your hardware</li>
                <li>You're comfortable configuring the BIOS based on instructions.</li>
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,559 @@
+    Please note that these installation methods, while they provide some
+    columnsibility for advanced users, they may also have some major limitations
+    on usage of Add-ons and other important Home Assistant features. Compare
+    installation methods here.
```
```suggestion
    Note that while these installation methods may provide some
    features for advanced users, they may also have some major limitations.
    For example, add-ons and other important Home Assistant features may not be available. Compare
    installation methods here.
```

c0ffeeca7 reviewed (commented):
minor suggestions. Not sure the text fits the space.

c0ffeeca7 committed:
Apply suggestions from code review

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,561 @@
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>You're comfortable doing the following tasks based on instructions:</li>
```
```suggestion
                <li>You're comfortable following instructions on:</li>
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,561 @@
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>You're comfortable doing the following tasks based on instructions:</li>
```
```suggestion
                <li>You're comfortable following instructions on:</li>
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,561 @@
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>You're comfortable doing the following tasks based on instructions:</li>
+                <li>Writing boot images to your boot medium</li>
```
```suggestion
                <li>Writing boot images</li>
```

c0ffeeca7 committed:
shorten text to fit

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,561 @@
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Interest in setting up a smart home</li>
```
```suggestion
                <li>Interest in setting up a smart home</li>
                <li>Know how to find an IP address on your router</li>
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,561 @@
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Assembling a Raspberry Pi setup</li>
+                <li>Flashing a Raspberry Pi</li>
```
```suggestion
                <li>Flashing a Raspberry Pi</li>
                <li>Know how to find an IP address on your router</li>
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,561 @@
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>You're comfortable following instructions on:</li>
```
```suggestion
                <li>Know how to find an IP address on your router</li>
                <li>You're comfortable following instructions on:</li>
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,561 @@
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>You're comfortable following instructions on:</li>
```
```suggestion
                <li>Know how to find an IP address on your router</li>
                <li>You're comfortable following instructions on:</li>
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,561 @@
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Advanced knowledge of Windows</li>
```
```suggestion
                 <li>Know how to find an IP address on your router</li>
                <li>Advanced knowledge of Windows</li>
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,561 @@
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Advanced knowledge of Windows</li>
```
```suggestion
                <li>Know how to find an IP address on your router</li>
                <li>Advanced knowledge of Windows</li>
```

c0ffeeca7 reviewed (commented):
add prereq that you need to know how to find an IP address on your router. Input by cogneato.

c0ffeeca7 committed:
Apply suggestions from code review

bramkragten committed:
fix columns

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,567 @@
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Interest in setting up a smart home</li>
+                <li>Know how to find an IP address on your router</li>
```
```suggestion
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,567 @@
+              <ul>
+                <li>Assembling a Raspberry Pi setup</li>
+                <li>Flashing a Raspberry Pi</li>
+                <li>Know how to find an IP address on your router</li>
```
```suggestion
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,567 @@
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Know how to find an IP address on your router</li>
```
```suggestion
```

c0ffeeca7 commented on source/installation/index.html:
```diff
@@ -6,263 +6,567 @@
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Know how to find an IP address on your router</li>
```
```suggestion
```

c0ffeeca7 committed:
remove prereq on knowing how to find an IP on router.

- if all goes as designed, the instance is recognized automagically
- if not, this is a troubleshooting case

bramkragten committed:
Update index.html

bramkragten committed:
update images

bramkragten committed:
update icons

frenck reviewed (approved):
Let's go!

Thanks, @bramkragten & @c0ffeeca7!

../Frenck
